### DISCUSSION TITLE
[Feature] Back arrow goes to image view, if appropriate

### DISCUSSION THREAD

**Discussion by bcsteeve** [immich-app/immich#3467] [Feature Request]
### The feature

Picture this:

I'm in the main photos display, or let's say an album.  I've clicked on a photo and I'm in the detailed view.  I click on a recognized face and it takes me to that face's view.  Now I click the "back" arrow and I would _expect_ to go back to where I was, which was the detailed view of the image I was on.

Instead, I'm taken to the _top_ of the main photos page or album that I was in.  That then means I lose my place of where I was.  Can be very frustrating if I'm going through a large album of photos one by one (in this case, I was checking that faces were correctly tagged in my "favorites").

It should take me back to where I was, not the general area.

Make sense?  This might be a "bug", but I wasn't sure.

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by alextran1502** [comment]
You can follow this issue for update https://github.com/immich-app/immich/issues/2906
